src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsGroupComponent.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import org.broadinstitute.dsde.rawls.model.{RawlsGroupShort, RawlsGroupEmail, RawlsGroupName}
+
+trait RawlsGroupComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class RawlsGroupTable(tag: Tag) extends Table[RawlsGroupShort](tag, "GROUP") {
+    def groupName = column[String]("groupName", O.PrimaryKey)
+    def groupEmail = column[String]("groupEmail")
```

Comment:
Yes.  `RawlsUserRef` uses `groupName`, so I changed it for consistency with the existing code.